src/libs/antares/study/study.cpp

```diff
@@ -773,51 +779,76 @@ bool Study::prepareOutput()
     }
 
     logs.info() << "  Output folder : " << folderOutput;
-
-    // Settings
-    // TODO : use writer
-    buffer.clear() << folderOutput << SEP << "about-the-study";
-    IO::Directory::Create(buffer);
-
-    // TODO : use writer
-    if (not simulation.saveToFolder(buffer))
-        return false;
-
-    // TODO : use writer
-    // Write the header as a reminder too
-    buffer.clear() << folderOutput << SEP << "about-the-study" << SEP << "study.ini";
-    header.saveToFile(buffer, false);
-
-    // copying the generaldata.ini
-    buffer.clear() << folder << SEP << "settings" << SEP << "generaldata.ini";
-    String dest;
-    // TODO : use writer
-    dest << folderOutput << SEP << "about-the-study" << SEP << "parameters.ini";
-    if (IO::errNone != IO::File::Copy(buffer, dest))
-        logs.error() << "impossible to copy " << dest;
-
-    // antares-output.info
-    buffer.clear() << folderOutput << SEP << "info.antares-output";
-    IO::File::Stream f(buffer, IO::OpenMode::write | IO::OpenMode::truncate);
-    if (f.opened())
-    {
-        String nowstr;
-        DateTime::TimestampToString(nowstr, "%Y.%m.%d - %H:%M", now);
-        f << "[general]";
-        f << "\nversion = " << (uint)Data::versionLatest;
-        f << "\nname = " << simulation.name;
-        f << "\nmode = " << StudyModeToCString(parameters.mode);
-        f << "\ndate = " << nowstr;
-        f << "\ntitle = " << nowstr;
-        f << "\ntimestamp = " << now;
-        f << "\n\n";
-    }
-    else
-        logs.error() << "I/O: impossible to write " << buffer;
-
     return true;
 }
 
+void Study::saveMiscFilesIntoOutput()
+{
+  String path;
+  path.reserve(1024);
+
+  path.clear() << "about-the-study";
+  simulation.saveUsingWriter(pResultWriter, path);
```

Comment:
As said in **simulation.h**'s review, we should rename study's **simulation** data member into **user_notes_**

Reply:
I've renamed it to `SimulationMetadata`

Reply:
I'm sorry but **simulation meta data** doesn't speak to me. I still think **user's notes** is better. 
Question : how is it called in Antares Web ?
We should also change the name of the source file (simulation.h is unrelated to what this class does)

Reply:
Renamed to `simulationComments`